output/index.yaml

```diff
@@ -0,0 +1,110 @@
+openapi: 3.0.0
+info:
+  title: Sentences API
+  description: Sentences REST API
+  version: 0.1.1
+servers:
+- url: https://virtserver.swaggerhub.com/MHAMLIN/sentences/0.1.1
+  description: SwaggerHub API Auto Mocking
+paths:
+  /sentences:
+    get:
+      description: Returns a list of sentences
+      responses:
+        "200":
+          description: Successful response
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/Sentence'
+components:
+  schemas:
+    Sentence:
+      required:
+      - id
+      type: object
+      properties:
+        id:
+          type: string
+          description: Unique identifier for the sentence
+          format: uuid
+        state:
+          pattern: "^\\d{2}"
+          type: string
+          description: State FIPS code
+        county:
+          pattern: "^\\d{5}"
+          type: string
+          description: County FIPS code
+        date:
+          type: string
+          description: "Sentencing date (month, day, and year)"
+          format: date
+        fine:
+          $ref: '#/components/schemas/SentenceTypeWithFines'
```

Comment:
I like what you did to combine the duration and type together - however I am slightly concerned that unpacking these responses might be more difficult and it could be better to store the boolean part at the top level of the response. I am open to either though

Reply:
I think it would be easier to talk through this one in our meeting.